Phase01/Phase01-StudentAverage/Student.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Phase01_StudentAverage
+{
+    internal class Student
+    {
+        public Student(int studentNumber, string firstName, string lastName)
```

Comment:
use field initialization

Reply:
Field initialization is used to ensure our fields are never null, but here we have a constructor which prevent this itself. so this change seems to be unnecessary. but we applied it anyway.